lib/helper.py

```diff
@@ -154,3 +155,14 @@ def install_packages(package_list):
             logger.debug(output)
             not_installed = True
     return not_installed
+
+
+def copy_dir_file(src, dest):
+    """
+    Copy all files from one dir to other dir
+    """
+    for item in os.listdir(src):
+        s_file = os.path.join(src, item)
+        d_file = os.path.join(dest, item)
+        shutil.copy2(s_file, d_file)
+    logger.debug("coped all files from  %s to %s" % (src, dest))
```

Comment:
@PraveenPenguin can this be `logger.info()` or the removing scripts part be `logger.debug()`. I want both to be similar.

Reply:
logger.info will be right will adopt this